hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -89,8 +89,15 @@ export class Insights extends React.Component<Record<string, object>, IInsightsS
     // selected buckets value should be reset to all buckets
     let selectedBuckets = [allBucketsOption];
     // Update bucket options only if one volume is selected
-    if (selectedVolumes && selectedVolumes.length === 1) {
-      const selectedVolume = selectedVolumes[0].value;
+    if (selectedVolumes && selectedVolumes.length !== null && (selectedVolumes.length === 2 && selectedVolumes[0].value == '*') || (selectedVolumes.length === 1)){
```

Comment:
just curious, what does the * mean? and why it's the first element in selectedVolumes?

Reply:
const allVolumesOption: IOption = {
  label: 'All Volumes',
  value: '*'
};

When we are selecting only one Volume then All volumes is getting selected by Default it is natural behaviour of dropdown so we are checking condition for One volume and multiple volume.

Reply:
i see!! Thanks!